CHANGELOG.md

```diff
@@ -149,7 +149,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (x/staking) [#12973](https://github.com/cosmos/cosmos-sdk/pull/12973) Removed `stakingkeeper.RandomValidator`. Use `testutil.RandSliceElem(r, sk.GetAllValidators(ctx))` instead.
 * (x/gov) [#13160](https://github.com/cosmos/cosmos-sdk/pull/13160) Remove custom marshaling of proposl and voteoption. 
 * (types) [#13430](https://github.com/cosmos/cosmos-sdk/pull/13430) Remove unused code `ResponseCheckTx` and `ResponseDeliverTx`
+* (auth) [#13460](https://github.com/cosmos/cosmos-sdk/pull/13460) The `q auth address-by-id` CLI command has been renamed to `q auth address-by-acc-num` to be more explicit. However, the old `address-by-id` version is still kept as an alias, for backwards compatibility.
```

Comment:
Wh are there so many unrelated changelog modifications? Was this due to a bad merge/rebase?

Reply:
It seems some valid changelogs get removed in this commit on main branch: https://github.com/cosmos/cosmos-sdk/commit/9d9f7df3d6e734ae9b806716ab1c4a18d3946237

Reply:
Changelog entry are deleted from unreleased when backported to the latest released version. As effectively it isn't nee change anymore if it exist in older versions. Please do not re-add them.